.github/workflows/release.yaml

```diff
@@ -0,0 +1,141 @@
+              RAW_VERSION=${{ steps.get_latest_release.outputs.tag }}
+              CLEAN_VERSION=${RAW_VERSION#v}
+            else 
+              CLEAN_VERSION=${{ steps.get_latest_release.outputs.version }}
+            fi
+            echo "VERSION=$CLEAN_VERSION" >> $GITHUB_OUTPUT
+            echo "RELEASE=${{ steps.get_latest_release.outputs.release }}" >> $GITHUB_OUTPUT
+            echo "TAG=${{ steps.get_latest_release.outputs.tag }}" >> $GITHUB_OUTPUT
+          else
+            RAW_VERSION=${{ github.event.release.tag_name }}
+            CLEAN_VERSION=${RAW_VERSION#v}
+            echo "VERSION=$CLEAN_VERSION" >> $GITHUB_OUTPUT
+            echo "RELEASE=1" >> $GITHUB_OUTPUT
+            echo "TAG=$RAW_VERSION" >> $GITHUB_OUTPUT
+          fi
+
+  create-rpm:
+    needs: gather-informations
+    uses: alumet-dev/packaging/.github/workflows/build_rpm.yaml@ci/create-rpm-package  # TODO CHANGE !!!
+    with:
+      target-architecture: x86_64
+      build-version: ${{ needs.gather-informations.outputs.version }}
+      release-version: ${{ needs.gather-informations.outputs.release }}
+      tag: ${{ needs.gather-informations.outputs.tag }}
+
+  attach-artifacts:
+    needs:
+      - create-rpm
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Download RPM artifact
+        uses: actions/download-artifact@v4
+        with:
+          pattern: '**/*.rpm'
+          path: ./rpm
+
+      - name: Release
+        if: ${{ github.event_name != 'workflow_dispatch' }}
+        uses: softprops/action-gh-release@v2
+        with:
+          files: |
+            rpm/**/*.rpm
+
+      - name: Get latest release and delete old artifacts
+        if: ${{ github.event_name == 'workflow_dispatch' }}
+        id: get_latest_release_and_delete_old_artifacts
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const latestRelease = await github.rest.repos.getLatestRelease({
+              owner: context.repo.owner,
+              repo: context.repo.repo
+            });
+
+            let id = latestRelease.data.id;
+            let tag = latestRelease.data.tag_name;
+
+            const assets = await github.rest.repos.listReleaseAssets({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              release_id: id
+            });
+
+            for (const asset of assets.data) {
+              await github.rest.repos.deleteReleaseAsset({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                asset_id: asset.id
+              });
+            }
+            console.log(`Release id: ${id}`);
+            console.log(`Release tag: ${tag}`);
+            core.setOutput("release_id", id);
+            core.setOutput("release_tag", tag);
+
+      - name: Upload new artifacts to github release
```

Comment:
"Upload RPM artifacts to github release"
cause we won't use this step it for every artifacts

Reply:
also this step is always executed but use `get_latest_release_and_delete_old_artifacts.release_tag` output how does it works in case ` github.event_name != 'workflow_dispatch'` ?
maybe you should add an if parameter to make explicit that it's the update release case.